### PR TITLE
Update Donate

### PR DESCRIPTION
**Pull request by rbxbx**
https://github.com/datafruits/datafruits/issues/694

### CODE REVIEW

File: app/components/dj-donate-button.hbs [hunk 1]
```diff
@@ -1,3 +1,4 @@
-<a href={{donationLink}} ...attributes class="donate-button" target="_blank" rel="noopener">
+<a {{action "toggleDjDonateModal" }} ...attributes class="donate-button">
```
Comment (mcfiredrill) [inline app/components/dj-donate-button.hbs]: This is the new syntax for actions :hammer_and_wrench: 
```suggestion
<a {{on "click" this.toggleDjDonateModal}} ...attributes class="donate-button">
```

File: app/components/dj-donate-modal.hbs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<a {{action "toggleDjDonateModal" }} ...attributes class="donate-button">
```
Comment (mcfiredrill) [inline app/components/dj-donate-modal.hbs:1]: Not a big deal but this is the new action syntax: :hammer_and_wrench: 
```suggestion
<a {{on "click" this.toggleDjDonateModal}} ...attributes class="donate-button">
```

Comment (rbxbx) [reply app/components/dj-donate-modal.hbs]: Noted and fixed, thanks!

**Comment by mcfiredrill** [comment]
Does this failing test need to be removed ?
```
not ok 18 Chrome 91.0 - [87 ms] - Integration | Component | dj-donate-button: it renders
    ---
        actual: >
            [object Object]
        stack: >
            Error: Compile Error: Cannot find component dj-donate-button

```

**Comment by rbxbx** [comment]
> Does this failing test need to be removed ?
> 
> ```
> not ok 18 Chrome 91.0 - [87 ms] - Integration | Component | dj-donate-button: it renders
>     ---
>         actual: >
>             [object Object]
>         stack: >
>             Error: Compile Error: Cannot find component dj-donate-button
> ```

good catch, thanks!

File: app/components/dj-donate-modal.hbs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        >
+        X
+      </button>
+    </div>
+
+    <div class="modal-body m-1">
+      You can donate to the DJ here:
+      <a href={{donationLink}} target="_blank" rel="noopener"> {{donationLink}} </a>
+      <br />
+      Or you can also support us on <a href="https://patreon.com/datafruits" target="_blank" rel="noopener"> patreon </a> or with a one time donation on <a href="https://paypal.me/datafruitsfm" target="_blank" rel="noopener"> paypal. </a>
```
Comment (mcfiredrill) [inline app/components/dj-donate-modal.hbs:22]: We should use i18n for these strings but I can always add them later

**Comment by mcfiredrill** [comment]
Deployed here if anyone wants to check it out https://datafruits-review-pr-698.herokuapp.com/chat

**Comment by mcfiredrill** [comment]
LGTM I'll merge and we can make the translations later if that's cool :shrimp:

**Comment by rbxbx** [comment]
> LGTM I'll merge and we can make the translations later if that's cool 🦐

works for me if you're okay with that! 🍥

**Pull request merged**